models/service.go

```diff
@@ -61,8 +61,19 @@ type ServiceOverview struct {
 	Health ServiceHealth `json:"health,omitempty"`
 }
 
+type ClusterServices struct {
+	// Cluster where the services live in
+	// required: true
+	// example: east
+	Cluster string `json:"cluster"`
+	// Services list for namespaces of a single cluster
+	// required: true
+	Services    []ServiceOverview `json:"services"`
```

Comment:
I see that you have renamed `ServiceOverview` type to `ServiceListItem` in the frontend, but not in the backend. I think it is easier for developers that type names are aligned to avoid confusion.

Reply:
In frontend `ServiceListItem extends ServiceOverview` by adding few frontend list view specific properties for better ordering.